horace_core/sqw/page_operations/PageOpBase.m

```diff
@@ -63,6 +63,10 @@
         old_file_format
         %
         write_handle
+        % An algorithm applied to a sqw object with missing data range
+        % should issue warning that range is recalculated unless the
+        % algorithm is the algorithm, which recalculates missing range
+        do_missing_range_warning;
```

Comment:
Dislike properties/values with "verb-like" names. prefer `is` over `do` or simply `missing_range_warning` or `missing_range_warning_enabled`

Reply:
I would disagree. Pretty nice forceful statement